packages/x-date-pickers/src/locales/enUS.ts

```diff
@@ -70,6 +70,7 @@ const enUSPickers: PickersLocaleText<any> = {
   fieldYearPlaceholder: (params) => 'Y'.repeat(params.digitAmount),
   fieldMonthPlaceholder: (params) => (params.contentType === 'letter' ? 'MMMM' : 'MM'),
   fieldDayPlaceholder: () => 'DD',
+  fieldWeekDayPlaceholder: (params) => (params.contentType === 'letter' ? 'DDDD' : 'DD'),
```

Comment:
@gerdadesign what placeholder should we put for the week day ? 
It can be either in a letter form (`Friday`, `Fri`) or in a digit for (`05`, `5`).
It's very similar to the month so I took the same approach.
But the `D` is already used and I don't have an alternative idea.

Reply:
@flaviendelangle  Can you elaborate with an example of this in context? 

What's the difference between Day and WeekDay, especially if it can be in letter or digit form?

Reply:
Ahh, ok. thank you for the clarification. 

I'm seeing a couple sources that use `EEEE` (even though that's not super intuitive). 
[Unicode](http://www.unicode.org/reports/tr35/tr35-31/tr35-dates.html#Date_Format_Patterns),  [Oracle](https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html) 

There's also the convention of distinguishing between capital and lowercase: using `D` or `DD` for the day of the month (**25th** of December) and using `ddd` for the shortened day of the week (Tue, Wed, Thu) and `dddd` for the long day of the week (Tuesday, Wednesday, Thursday). `d` for representing this in numeric format. 
[Example 1](https://day.js.org/docs/en/display/format), [This one](https://www.php.net/manual/en/datetimeimmutable.createfromformat.php) flips the capitalization convention. 

Since we already distinguish different fields based on capitalization (`M` for month vs `m` for minute), perhaps we can continue to this pattern with `DD` vs `dddd`? 
Though ultimately, it seems the best practice is to use today's date as an example placeholder. Would users be able to do that?

Reply:
> Though ultimately, it seems the best practice is to use today's date as an example placeholder. Would users be able to do that?

Do something like `Friday dd of MMMM` to take the example above ?
Yes I don't see anything blocking.

Reply:
I wouldn't mix methods, but it would be nice to allow users to choose how they prefer placeholder values. 

So if I need to enter **Friday, December 16, 2022**:

Placeholder with abbreviations:  `dddd, MMMM DD, YYYY` 
OR 
Placeholder with written descriptions: `Weekday, Month Day, Year`
OR 
Placeholder with today's date as an example of format:  `Tuesday, January 10, 2023`

Generally, it seems the last option is best practice, but someone may choose a different placeholder option in different contexts.

Reply:
Okay
Each one of them is fairly easy to achieve